### ISSUE TITLE
[PE-D][Tester D] Deadline is before today

### ISSUE THREAD

**Issue by nus-pe-script** [AY2324S1-CS2103T-W08-3/tp#145]
Jobby allows me to change the deadline to a day before today. Maybe a validation check could be done to the last updated so as to see if its a valid deadline 

![Screenshot 2023-11-03 at 9.33.51 PM.png](https://raw.githubusercontent.com/mingyu-wan/ped/main/files/e2d4df2d-8550-4939-96e8-dd5c46edf8bc.png)

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: mingyu-wan/ped#7

**Comment by wamps-jp** [comment]
I don't believe this is an issue with functionality - we could instead mention it in the UG, that you're allowed to change the deadline to a date before the current. Blocking that functionality wouldn't make sense.
